client/capacitor/package.json

```diff
@@ -13,27 +13,7 @@
     "build": "webpack",
     "preview": "webpack serve"
   },
-  "dependencies": {
-    "@capacitor/android": "^7.4.1",
-    "@capacitor/assets": "^3.0.5",
-    "@capacitor/camera": "latest",
-    "@capacitor/core": "^7.2.0",
-    "@capacitor/device": "^7.0.1",
-    "@capacitor/ios": "^7.4.1",
-    "@capacitor/splash-screen": "latest"
-  },
   "devDependencies": {
-    "@babel/core": "^7.27.3",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.27.2",
-    "@capacitor/cli": "^7.2.0",
-    "babel-loader": "^10.0.0",
-    "html-webpack-plugin": "^5.6.3",
-    "path": "^0.12.7",
-    "webpack": "^5.99.9",
-    "webpack-cli": "^6.0.1"
-  },
-  "author": "",
-  "license": "ISC"
-}
+    "@capacitor/cli": "^7.2.0"
```

Comment:
Please update `package-lock.json`, otherwise this won't get installed to `node_modules`